website/docs/r/synapse_private_link_hub.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Synapse"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_synapse_private_link_hub"
+description: |-
+  Manages a Synapse Private Link Hub.
+---
+
+# azurerm_synapse_private_link_hub
+
+Manages a Synapse Private Link Hub.
+
+## Example Usage
+
+```hcl
+resource "azurerm_synapse_private_link_hub" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Synapse Private Link Hub. Changing this forces a new Synapse Private Link Hub to be created.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Synapse Private Link Hub.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the Synapse Private Link Hub.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
+
+* `create` - (Defaults to 30 minutes) Used when creating the Synapse Private Link Hub.
+* `read` - (Defaults to 5 minutes) Used when retrieving the Synapse Private Link Hub.
+* `update` - (Defaults to 30 minutes) Used when updating the Synapse Private Link Hub.
+* `delete` - (Defaults to 30 minutes) Used when deleting the Synapse Private Link Hub.
+
+## Import
+
+Synapse Private Link Hub can be imported using the `resource id`, e.g.
+
+```shell
+terraform import azurerm_synapse_private_link_hub.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.Synapse/privateLinkHubs/privateLinkHub1
```

Comment:
We should double check that this format is correct once the id template is added to `resourceids.go`